### PR TITLE
#38 dungeon indicator

### PR DESCRIPTION
**Pull request by franomai**
Still need to fix the indicator on fizzle load, but the core mod doesn't fall over now with a null mod.

### CODE REVIEW

File: mod/cornercutter/ModLoader/CutterConfig.cs [hunk 1]
```diff
@@ -61,6 +62,7 @@ public void LoadGlobalSettings()
             string settingsLocation = Path.GetFullPath(Path.Combine(cornercutterFolder, "settings.json"));
             GlobalInfoDTO settings = Reader.ReadGlobalInfo(settingsLocation);
             GlobalOptions = settings.GlobalOptions;
+            HasCurrentMod = (settings.CurrentModFilename != null);
```
Comment (pumbas600) [inline mod/cornercutter/ModLoader/CutterConfig.cs:65]: Is this any particular reason this is in brackets?

File: mod/cornercutter/ModFeature/ModSetting/StartingSkillSwapper.cs [hunk 2]
```diff
@@ -11,14 +11,14 @@ class StartingSkillSwapper
     {
         static void Postfix(ref DungeonManager __instance)
         {
-            if (!CutterConfig.Instance.CornercutterIsEnabled()) return;
+            CutterConfig cornercutter = CutterConfig.Instance;
+            if (!(cornercutter.CornercutterIsEnabled() && cornercutter.HasCurrentMod)) return;
 
             int currentFloor = __instance.currentFloor + 1;
             WeightedSkill[] startingSkills = CutterConfig.Instance.StartingSkills;
```
Comment (pumbas600) [inline mod/cornercutter/ModFeature/ModSetting/StartingSkillSwapper.cs:18]: You can replace `CutterConfig.Instance` with `cornercutter` here.

**Review by pumbas600 — CHANGES_REQUESTED** [review]
Looks good to me, just requested one small change :)

File: mod/cornercutter/ModLoader/CutterConfig.cs (hunk 1 above)
Comment (franomai) [reply mod/cornercutter/ModLoader/CutterConfig.cs:65]: for fun

File: mod/cornercutter/ModFeature/ModSetting/StartingSkillSwapper.cs (hunk 2 above)
Comment (franomai) [reply mod/cornercutter/ModFeature/ModSetting/StartingSkillSwapper.cs:18]: good catch, fixed

**Review by pumbas600 — APPROVED** [review]
Looks good to me!

**Pull request merged**